stacks/environment-azure/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "password" {
+    value = random_password.password.result
```

Comment:
I'm not entirely sure I understand what this is for, but typically we'd add `sensitive = true` so this value doesn't get printed in your shell in plaintext when running `terragrunt apply`.

Reply:
Under review with team